README.md

```diff
@@ -98,12 +98,12 @@ Our suggested method for quick deployment is Heroku's one-click option:
 
 However, PostHog can be deployed anywhere you want! Here are step-by-step tutorials we've written for deployment options using all the major cloud providers:
```

Comment:
cc @tiina303 worth updating this for clickhouse as well.

Reply:
#5237 does this - I'll address the comments in it now